pywbem/cim_obj.py

```diff
@@ -1391,86 +1576,111 @@ def from_wbem_uri(wbem_uri):
         WBEM URI string.
 
         The WBEM URI string must be a CIM instance path in untyped WBEM URI
-        format, as defined in :term:`DSP0207`.
+        format, as defined in :term:`DSP0207`, with these extensions:
+
+        * DSP0207 restricts the namespace types (URI schemes) to be one of
+          ``http``, ``https``, ``cimxml-wbem``, or ``cimxml-wbems``. Pywbem
+          tolerates any namespace type, but issues a :exc:`py:UserWarning` if
+          it is not one of the namespace types defined in DSP0207.
+
+        * DSP0207 requires a slash before the namespace name. For local WBEM
+          URIs (no namespace type, no authority), that slash is the first
+          character of the WBEM URI. For historical reasons, pywbem tolerates a
+          missing leading slash for local WBEM URIs. Note that pywbem requires
+          the slash (consistent with DSP0207) when the WBEM URI is not local.
+
+        * DSP0207 requires datetime values in keybindings to be surrounded by
+          double quotes. For historical reasons, pywbem tolerates datetime
+          values that are not surrounded by double quotes, but issues a
+          :exc:`py:UserWarning`.
+
+        * DSP0207 does not allow the special float values INF, -INF, and NaN
+          in WBEM URIs (according to realValue in DSP0004). However, the
+          CIM-XML protocol supports representation of these special values,
+          so to be on the safe side, pywbem supports these special values as
+          keybindings in WBEM URIs.
+
+        Note that keybindings that are references are supported, recursively.
```

Comment:
No real reason for the "Note that" phrases.  I tend to do the same thing but realize later that not really necessary

Reply:
DONE: Removed "Note that" in this place and a few lines down.